GliaWidgets/Component/Bubble/BubbleStyle.Accessibility.swift

```diff
@@ -0,0 +1,24 @@
+extension BubbleStyle {
+    /// Accessibility properties for BubbleStyle.
+    public struct Accessibility: Equatable {
```

Comment:
Why do they need to conform to Equatable protocol?

Reply:
[For XCTAssertEqual](https://github.com/salemove/ios-sdk-widgets/pull/253#discussion_r845776310).